tests/scripts/format.py

```diff
@@ -185,6 +185,9 @@ def main(argv):
         print("7. Unzip the patch file.")
         print("8. git apply format.patch")
 
+    if (returncode != 0) and ("AzureDevOps" in os.environ) and (os.environ["AzureDevOps"] == "1"):
```

Comment:
Can we use `TF_BUILD` (see https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml) instead of introducing another env.variable?

Reply:
cc @sbomer

Reply:
Thanks @echesakovMSFT, I changed the PR as you suggested.